From b887010c03029b9d42848bec3f9ecb1c51ee12f3 Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Wed, 26 Oct 2016 16:38:03 +0300
Subject: [PATCH 0560/1345] dts: a7040: enable phy initialization for GOP2

commit  3a51c95ed24b032c5a109509c04bbf342dbbb178 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

- GOP2 connected to PHY2 with SMI address 0.
- Without phy configuration phy cannot be connected in Linux.

Change-Id: I9183919ac952359399dd49616835f2a457152c57
Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/33372
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/marvell/armada-7040-db-all-pcie.dts   |    1 +
 .../boot/dts/marvell/armada-7040-db-router.dts     |    1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db-all-pcie.dts b/arch/arm64/boot/dts/marvell/armada-7040-db-all-pcie.dts
index 3691bf0..ec2c2c8 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db-all-pcie.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db-all-pcie.dts
@@ -130,6 +130,7 @@
 				};
 				emac2: mac2 {
 					phy-mode = "rgmii"; /* rgmii-0 */
+					phy = <&phy2>;
 				};
 				emac3: mac3 {
 					phy-mode = "rgmii"; /* rgmii-1 */
diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db-router.dts b/arch/arm64/boot/dts/marvell/armada-7040-db-router.dts
index f0172cd..43fdc3a 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db-router.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db-router.dts
@@ -130,6 +130,7 @@
 				};
 				emac2: mac2 {
 					phy-mode = "sgmii"; /* lane-0 */
+					phy = <&phy2>;
 				};
 				emac3: mac3 {
 					phy-mode = "rgmii"; /* rgmii-1 */
-- 
1.7.9.5

